services/core-api/app/api/projects/information_requirements_table/models/information_requirements_table.py

```diff
@@ -80,6 +84,7 @@ def find_by_mine_document_guid(cls, mine_document_guid):
         except ValueError:
             return None
 
+    @classmethod
     def send_irt_submit_email(self):
         project_lead_email = self.project.project_lead.email if self.project.project_lead else None
         recipients = [MAJOR_MINES_OFFICE_EMAIL, project_lead_email
```

Comment:
wait, if this is a classmethod, then shouldn't the first parameter of the function be the class rather than the instance?